.github/ISSUE_TEMPLATE/GC-Articles-Publishing

```diff
@@ -0,0 +1,57 @@
+---
+name: GC Articles Publishing User Story Template
+about: Template for creating new User Stories for Notify backlog
+title: 'Give your user story a simple, clear title'
+labels: ''
+assignees: ''
+
+---
+## Description
+As a (user), I need to be able to do (X) so that I can achieve (Y) outcome. 
+
+WHY are we building?
+WHAT are we building?
+VALUE created by our solution
+
+## Documentation and Artifacts
+Good docs, figma mockups, ADRs, screenshots etc.
+
+## Acceptance Criteria
+Given some context, when (X) action occurs, then (Y) outcome is achieved 
+
+- [ ] Cypress UI tests if needed.
+- [ ] Generate appropriate log messages so that executions of this feature can be tracked
+- [ ] Can misuse of this feature cause harm? If yes, create an alert
+- [ ] Update the status of related findings, insights, and hypotheses on the Research Airtable
+- [ ] Once change/fix/feature is implemented, link relevant Airtable records to design artifacts (Figma)
+
+* A11y
+* Bilingualism
+* Privacy considerations
+* Security controls in place
+* Measuring success and metrics
+
+## Related [Research Airtable](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwQUX3tUxbMxYS7C?blocks=hide) records
+- (find stuff yourself)
+
+## QA Steps
+- [ ] Tested in a realistic production scenario
+
+## GC Articles Publish checklist
+- [ ]  Supporting feature is released and unflagged
```

Comment:
This might not be needed here if we have a dedicated template for GCA. Likely the feature would have been already released? We could also keep it if we think we might have concurrent GCA and story cards open.

Reply:
Good consideration. I think we can keep it in there still in case we have a related feature being released alongside a new page or updated content in GCA